Source/ACE.Entity/Character.cs

```diff
@@ -112,6 +112,8 @@ public uint Level
             set { propertiesInt[PropertyInt.Level] = value; }
         }
 
+        public uint AvailableSkillCredits { get; set; }
```

Comment:
To allow the addition of skill credits, I created a public accessor for AvailableSkillCredits.